pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/UnnecessaryImport.xml

```diff
@@ -157,7 +158,8 @@ public class Foo {
         ]]></code>
     </test-code>
 
-    <test-code>
+    <test-code regressionTest="false">
```

Comment:
I disabled this test, as we now need a classpath to correctly identify imported static methods/fields. This should make the rule more precise when the classpath is full, but maybe more prone to FPs when the classpath is improperly configured.

Reply:
I think, that's fine. At some point we need to require that PMD is called properly...